packages/checkout/core/src/services/sessions/session.utils.ts

```diff
@@ -0,0 +1,52 @@
+import { Session, SessionNormalized } from './sessions.types'
+
+export const normalizePaymentSession = (
+  paymentSession: Session,
+): SessionNormalized => {
+  const paymentSessionNormalized: SessionNormalized = {
+    ...paymentSession,
+    transactionConfig: null,
+    checkoutPaymentMethods: null,
+  }
```

Comment:
Acho que aqui nesse objeto já daria para iniciar ele e já processar os dados aqui, por exemplo:

```suggestion
  const paymentSessionNormalized: SessionNormalized = {
    ...paymentSession,
    transactionConfig: parseTransactionConfig(paymentSession),
    checkoutPaymentMethods: parsePaymentMethods(paymentSession),
  }
```

Aí poderia ser criado pequenas funções nesse mesmo file que fazem esse parse, o código fica com uma pegada mais funcional e imutável